data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,283 @@ mission "Arachi Cartoonist 2"
+			`	Eventually, a sphere finds its way into a hole, which lights up red for the color of the scoring team. Much of the crowd jumps up and screams, though you can't quite tell if everyone is a particular fan of anyone on the red team or if they're cheering simply because a point was finally scored. The sphere is shot back out automatically from a tube close to the hole and is immediately back in play.`
+			`	As the game continues each team scores a few times, but the blue team pulls into the lead. When they score their fifth point, the crowd's reaction is the loudest you've seen yet. The entire blue team gathers and celebrates as the commentators begin speaking. The helpful Kimek with the translator excitedly explains to you what is happening.`
+			`	"The winning score, they have reached! The winners, the blue team are!" they say, still in a cheering mood. After everyone settles down, the members of the teams are shown on a replay screen, starting with the yellow team, the lowest scoring, and progressing up to the blue team at the end. After this event, Ulmodi has risen to being placed third, and you notice that the Saryd who caught your eye in the forest earlier is in first place. After the individual scores are announced, the top twenty contestants celebrate by running laps around the arena.`
+			`	After the celebration has died down, the teams gather for a moment of sportsmanship before leaving the arena. At this point the crowd begins to file out, once more under Heliarch direction. You arrive back at where the train dropped you off and see that another one has arrived on the opposite side. The Heliarch agent directing your group explains that it is headed to the final arena where the cooperative game will take place, but that you are also free to travel back to the other arenas and view the rest of the other games.`
+				goto choice
+			
+			
+			label kimek
+			apply
+				set "coalition games: kimek"
+			`	The overly-packed train takes you to a savanna plain with a colossal cylindrical skyscraper, looking as if a stadium had been stretched vertically tenfold. Upon entering, you look up and see that, instead of a full ceiling for this level, there is only a large circular platform connected to the walls by thin beams. The underside of the platform has many individual small rings painted onto it, and a net just below that spans the entire room. Looking through the net, you can see that the same set up is repeated for the next level above, and each one above that, to the top of the tower.`
```

Comment:
```suggestion
			`	The overly-packed train takes you to a savanna plain with a colossal cylindrical skyscraper, looking as if a stadium had been stretched vertically tenfold. Upon entering, you look up and see that instead of a full ceiling for this level, there is only a large circular platform connected to the walls by thin beams. The underside of the platform has many individual small rings painted onto it, and a net just below that spans the entire room. Looking through the net, you can see that the same set up is repeated for the next level above, and each one above that, to the top of the tower.`
```